_topic_maps/_topic_map_osd.yml

```diff
@@ -1641,14 +1641,6 @@ Topics:
   Topics:
   - Name: Backup and restore by using VM snapshots
     File: virt-backup-restore-snapshots
-  - Name: Installing and configuring OADP
-    File: virt-installing-configuring-oadp
-  - Name: Backing up and restoring virtual machines
-    File: virt-backup-restore-overview
-  - Name: Backing up virtual machines
-    File: virt-backing-up-vms
-  - Name: Restoring virtual machines
-    File: virt-restoring-vms
 #  - Name: Collecting OKD Virtualization data for community report
```

Comment:
To keep this topic map in sync with `main` and to avoid spurious merge conflicts, please include the commented out books you included in the original PR.